controller/user.controller.js

```diff
@@ -49,11 +46,6 @@ const getUserExperiment = (user) => {
   return user.experiments;
 };
 
-const setCookie = () => {
-  const uuid = generateUuid();
```

Comment:
same, why delete?

Reply:
we don't need this for the same reason as the last one.